### ISSUE TITLE
commit-updated-env: Error: Input required and not supplied: token

### ISSUE THREAD

**Issue by pmrv** [pyiron/actions#82]
I'm running into an error, that I don't understand at all over in [potentialfit](https://github.com/pyiron/pyiron_potentialfit/actions/runs/7987993947/job/21811655211?pr=42).  The action ran before on other PRs, so I assumed the tokens are setup.  Did I break something somewhere?

**Comment by liamhuber** [comment]
In `pyiron_workflow` the update env step often failed for dependabot PRs. I just tracked down [one of these logs](https://github.com/pyiron/pyiron_workflow/actions/runs/7894097274/job/21544054294) and indeed it's the same token error. What always confused me is that `pyiron_contrib` is _also_ using the centralized CI and yet never seemed to run into this issue! To now make things completely insane, as of today (yesterday), the dependabot PRs on `pyiron_workflow` no longer seem to suffer from this problem. 

In conclusion: I have no idea what is going on.

I can, however, share my workaround, which has been to simply close and re-open the PR quickly. This seems to trigger something on the github side that runs the PR as though it's coming from me and not dependabot. Annoying, because the dependabot PRs then need a step of human intervention beyond clicking "merge", but only two extra clicks...so my motivation to fix the underlying issue has been minimal 😬

**Comment by liamhuber** [comment]
>  simply close and re-open the PR quickly

I just tried this on your behalf and this trick works fine on `potentialfit` too. The tests are still not passing, but you get past the env update no problem.

**Comment by pmrv** [comment]
Ok, weird.  Thanks for the work around.
